be2-scala/.github/workflows/be2-scala.yml

```diff
@@ -4,7 +4,7 @@ on:
   push:
     branches: [ work-be2-* ]
   pull_request:
-    branches: [ be2-scala ]
+    branches: [ master ]
```

Comment:
You want to move this file into the root `.github/workflows/ci.yml`

Reply:
Done! Didn't merge it with the common file since probably some things will change in the scala CI